_config.yml

```diff
@@ -33,3 +33,6 @@ linkedin_audience: 2842796
 twitter_user: trinodb
 twitter_hashtag: trinodb
 youtube_channel: trinodb
+
+paginate: 12
```

Comment:
So it's clear what this does:
> To enable pagination for posts on your blog, add a line to the _config.yml file that specifies how many items should be displayed per page:

```paginate: 5```

https://jekyllrb.com/docs/pagination/#enable-pagination

At first I thought this was total pages and we would have to keep updating the static value.